application/restore.sh

```diff
@@ -58,7 +41,7 @@ else
 fi
 
 start=$(date +%s)
-psql --host=${DB_HOST} --username=${DB_USER} --dbname=${DB_NAME} ${DB_OPTIONS}  < /tmp/${DB_NAME}.sql || STATUS=$?
+psql --host=${DB_HOST} --username=${DB_ROOTUSER} --dbname=postgres ${DB_OPTIONS}  < /tmp/${DB_NAME}.sql || STATUS=$?
```

Comment:
README still refers to `DB_NAME`. Is that OK?

Reply:
Yes.  DB_NAME is the name of the DB we want to restore which will be in the DB backup file (${DB_NAME}.sql).  The --dbname=postgres is required since the DB backup file DROPs $DB_NAME before creating it and you can't drop a DB that you are connected to.